### PR TITLE
fix: create release workflow

### PR DESCRIPTION
**Pull request by myishay**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Release Workflow
+on:
+  workflow_dispatch:
+    inputs:
+      name:
+        description: "Triggered by:"
```
Comment (dimabru) [inline .github/workflows/release.yml:6]: will github provide us info on who triggered the release without this input?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        required: true
+        default: "myishay"
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
```
Comment (dimabru) [inline .github/workflows/release.yml:13]: the step that triggers the travis build is missing here

File: .github/workflows/release.yml (hunk 1 above)
Comment (myishay) [reply .github/workflows/release.yml:6]: I didn't  find how to do it yet, but I will keep searching

Comment (myishay) [reply .github/workflows/release.yml:6]: found it! resolving this conversation

File: .github/workflows/release.yml (hunk 2 above)
Comment (myishay) [reply .github/workflows/release.yml:13]: added it now

Comment (myishay) [reply .github/workflows/release.yml:13]: @dimabru lmk what you think

**Review by dimabru — DISMISSED** [review]

**Pull request merged**